### ISSUE TITLE
Translation problem, but also on the source

### ISSUE THREAD

**Issue by paramecie** [rgriebl/brickstore#15]
I know it's a difficult point, but I propose to make it simple :-)

In various strings in English, there are either Lots or Items.

Sometimes both, like here (and this is wrong IMO):

![image](https://user-images.githubusercontent.com/32078594/103489743-c9fd4d80-4e16-11eb-883c-bdcdacf3834c.png)

Wrong as "Consolidate items" and then "Consolidate those 2 lots", then "item's attributes"...

Here it's another problem (it's the small Info window):

![image](https://user-images.githubusercontent.com/32078594/103489766-00d36380-4e17-11eb-8ff3-84a4c9bcd28e.png)

Because now we show for example "2 lots" while we used the "Add items" window :-(

And here those "lots" are all items now:
![image](https://user-images.githubusercontent.com/32078594/103489813-5740a200-4e17-11eb-8118-439fcd1e79cf.png)

OK, to make it simple, I would propose you (as BrickLink does in its Add items page) the following (in English, and I'll do this in French too):

### **1) In a general meaning, talk everywhere about Item(s) and never Lot(s).**

Example : Add Items, Consolidate Items...
To change in the Info widget:
Multiple items selected
Items (instead of lots)
Qty (or total qty, instead of items - as it is really the total qty!)

To change, also for example : 
"The quantities of %n lot(s)" => item(s)
"Would you like to consolidate those two lots" => items

### **2) Exceptions, and following BrickLink (Price Guide for example), say Lot when it's about lots for sale, lots in cart, lot ID:**

Show Lots for Sale on BrickLink... (this is correct)
Lot ID (correct)
%1 lots of your Shopping Cart could not be imported. (correct, it's lots in the Cart)

**In short - put item(s) everywhere apart for lots for sale, lots in cart, lot ID.**

Sorry, for the Bold and hoping it helps, but here it's a problem since 2006 in this translation (for me, IMHO).

THANK YOU!

**Comment by paramecie** [comment]
Also, here:
![image](https://user-images.githubusercontent.com/32078594/103491300-dbe4ed80-4e22-11eb-8327-24cb43f4fac2.png)

Comment and Remark should be singular, like Condition (otherwise should be Conditions).
And it affects only 1 item, not multiple items, so that would be singular.

**Comment by paramecie** [comment]
A few details:

![image](https://user-images.githubusercontent.com/32078594/103493055-6cc1c600-4e2f-11eb-8b95-caaef539ad4f.png)

As "Part #" has a space, "Pr. Orig.", "Qty.Orig" etc. should have also a space in between?

**Comment by paramecie** [comment]
This should be Price, as the column is Price, the value is used only in "Price Guide values" for example.

![image](https://user-images.githubusercontent.com/32078594/103493443-b27f8e00-4e31-11eb-941c-084368773245.png)

**Comment by ZZJHONS** [comment]
> A few details:
> 
> ![image](https://user-images.githubusercontent.com/32078594/103493055-6cc1c600-4e2f-11eb-8b95-caaef539ad4f.png)
> 
> As "Part #" has a space, "Pr. Orig.", "Qty.Orig" etc. should have also a space in between?

The thing is Qty.Orig And Pr.Orig. Normally have just a few characters so you have the columns shortened. But Part # can be more longer normally with decorated parts

**Comment by paramecie** [comment]
> The thing is Qty.Orig And Pr.Orig. Normally have just a few characters so you have the columns shortened. But Part # can be more longer normally with decorated parts

Sure, that was critical in 2005 ;-)

Of course it's maybe a bad example, but I guess you've now a 1920 px wide screen at the minimum, which is almost the double width we may have in 2005... What I mean is that I'm not sure 2 exctra chars are much a problem now. As you think of course.

**Comment by paramecie** [comment]
Also FYI Robert, I translated "Show" by "Open" (ouvrir, in French), like in "Show the BrickLink Catalog" as it opens a web page/tab.

At the contrary, "Show input errors" stays "Show" as it's showing internally, in the application.

**Comment by paramecie** [comment]
Also, "Remarks" and "Comments" in the document (columns) could be singular.
They're singular in the menu.
And no other column title has a plural.

And that would save 2 chars wide :-)

Maybe have the same everywhere at least?

**Comment by paramecie** [comment]
Minor but here the menu is "Import" and there's Import items in the Home page:

![image](https://user-images.githubusercontent.com/32078594/108661394-0589cf00-74cc-11eb-85cd-c5da5a924b68.png)

One or the other could be corrected?
